src/bundle/Controller/ContentTypeController.php

```diff
@@ -573,6 +620,47 @@ public function bulkDeleteAction(Request $request, ContentTypeGroup $group): Res
         return $this->redirect($this->generateUrl('ezplatform.content_type_group.view', ['contentTypeGroupId' => $group->id]));
     }
 
+    /**
+     * Handles removing draft content types based on submitted form.
+     *
+     * @throws TranslationInvalidArgumentException
+     * @throws InvalidOptionsException
+     * @throws \InvalidArgumentException
+     */
+    public function bulkDraftDeleteAction(Request $request, ContentTypeGroup $group): Response
+    {
+        $this->denyAccessUnlessGranted(new Attribute('class', 'delete'));
+        $form = $this->formFactory->deleteDraftContentTypes(
+            new DraftContentTypesDeleteData()
+        );
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted()) {
+            $result = $this->submitHandler->handle($form, function (DraftContentTypesDeleteData $data) {
+                foreach ($data->getContentTypes() as $contentTypeId => $selected) {
+                    $contentType = $this->contentTypeService->loadContentTypeDraft($contentTypeId);
+
+                    $this->contentTypeService->deleteContentType($contentType);
+
+                    $this->notificationHandler->success(
+                        $this->translator->trans(
+                            /** @Desc("Draft Content Type '%name%' deleted.") */
```

Comment:
```suggestion
                            /** @Desc("Content Type draft '%name%' deleted.") */
```
Actually I'd opt for "Content Type draft" to "draft (of) Content Type", but if it requires too many changes, please ignore this and the remaining comments.
But bapitals for Content Type everywhere, plz :)